server/src/domain/domain.config.ts

```diff
@@ -29,7 +29,8 @@ export const immichAppConfig: ConfigModuleOptions = {
     MACHINE_LEARNING_PORT: Joi.number().optional(),
     MICROSERVICES_PORT: Joi.number().optional(),
     SERVER_PORT: Joi.number().optional(),
-    DISABLE_CHECK_LATEST_VERSION: Joi.boolean().optional(),
+    CHECK_NEW_VERSION_INTERVAL: Joi.boolean().optional(),
+    DISABLE_CHECK_LATEST_VERSION: Joi.number().optional(),
```

Comment:
These could be combined into one by having `CHECK_NEW_VERSION_INTERVAL=0` mean that it is disabled.

Reply:
Great idea !